src/routes/path.ts

```diff
@@ -0,0 +1,12 @@
+export const AUTH = {
+  SIGN_IN: '/sign-in',
+  SIGN_UP: '/sign-up',
+} as const;
+
+export default ROUTE_PATH;
+  HOME: '/',
+  Recruit: '/recruit',
+  VISA_REGISTRATION: '/visa-registration',
+  POST_NOTICE: '/post-notice',
+  AUTH,
+} as const;
```

Comment:
이런 코드가 커밋이 되었네요...?

```suggestion
const ROUTE_PATH = {
  HOME: '/',
  Recruit: '/recruit',
  VISA_REGISTRATION: '/visa-registration',
  POST_NOTICE: '/post-notice',
  AUTH,
} as const;

export default ROUTE_PATH;
```

이렇게 쓰여야 하는게 아닐까요?